src/applications/claims-status/actions/index.jsx

```diff
@@ -147,16 +148,51 @@ export function fetchClaimsSuccess(response) {
   };
 }
 
-export function getClaimsV2() {
+const POLLING_INTERVAL = 1000;
+
+export function pollClaimsStatus({
+  onFailure,
+  onSuccess,
+  pollingInterval,
+  request = apiRequest,
+  shouldFail,
+  shouldSucceed
+}) {
+  return request(
+    '/evss_claims_async',
+    null,
+    response => {
+      if (shouldSucceed(response)) {
+        onSuccess(response);
+        return;
+      }
+
+      if (shouldFail(response)) {
+        onFailure(response);
+        return;
+      }
+
+      setTimeout(
```

Comment:
Noticed a lot of our polling implementations do timeout-first polling. Set up a poll first approach here.